hw2/05-form.html

```diff
@@ -7,13 +7,116 @@
+          <label for="email" class="col-12 col-form-label fw-bold"
+            >Email:</label
+          >
+          <div class="col-12">
+            <input type="email" class="form-control" id="email" name="email" />
+          </div>
+        </div>
+        <div class="form-group row my-2">
+          <label for="registration" class="col-12 col-form-label fw-bold"
+            >Registration Status:</label
+          >
+          <div>
+            <select
+              class="form-select"
+              aria-label="Default select example"
+              id="registration"
+              name="registration"
+            >
+              <option selected class="row">Choose an Option</option>
+              <option value="Freshmen" class="row">Freshmen</option>
+              <option value="Sophomore" class="row">Sophomore</option>
+              <option value="Junior" class="row">Junior</option>
+              <option value="Senior" class="row">Senior</option>
+            </select>
+          </div>
+        </div>
+
+        <div class="form-group row my-2">
+          <label for="courses" class="col-12 col-form-label fw-bold"
+            >have you taken any of the following courses?</label
+          >
+          <div id="courses">
+            <div class="form-check col-12">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                value="Programming Languages"
+                id="programmingLanguages"
+                name="courses"
+              />
+              <label class="form-check-label" for="programmingLanguages">
+                Programming Languages
+              </label>
+            </div>
+            <div class="form-check col-12">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                value="Operating Systems"
+                id="operatingSystems"
+                name="courses"
+              />
+              <label class="form-check-label" for="operatingSystems">
+                Operating Systems
+              </label>
+            </div>
+            <div class="form-check col-12">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                value="Full Stack Web Development"
+                id="fullStack"
+                name="courses"
+              />
+              <label class="form-check-label" for="fullStack">
+                Full Stack Web Development
+              </label>
+            </div>
+          </div>
+        </div>
+        <div class="form-group row my-2">
+          <label for="textarea" class="col-12 col-form-label fw-bold"
+            >Anything else?</label
+          >
+          <div>
+            <textarea class="form-control" id="textarea" rows="3"></textarea>
+          </div>
+        </div>
+        <div class="text-center row d-flex justify-content-center">
+          <div class="col">
```

Comment:
consider using d-flex class on the div encompassing the buttons to get  hem arranged like in the screenshot